Examples/security-template.ruleset

```diff
@@ -12,12 +12,15 @@ AWS::AmazonMQ::Configuration EngineVersion == 5.15.10 << Broker engine version s
 
 AWS::ApiGateway::DomainName SecurityPolicy == TLS_1_2 << API Gateway DomainName SecurityPolicy should use TLS 1.2
 
+AWS::ApiGatewayV2::DomainName DomainNameConfigurations.0.SecurityPolicy == TLS_1_2 << API GatewayV2 DomainName SecurityPolicy should use TLS 1.2
```

Comment:
Think `.0.` only checks the first `DomainNameConfiguration`

should probably check all of them

Reply:
Good call, I updated to be `DomainNameConfigurations.*.SecurityPolicy`

Thanks